src/main/java/com/exasol/adapter/dialects/exasol/ExasolSqlDialect.java

```diff
@@ -35,10 +35,11 @@ public class ExasolSqlDialect extends AbstractSqlDialect {
     public ExasolSqlDialect(final ConnectionFactory connectionFactory, final AdapterProperties properties) {
         super(connectionFactory, properties,
                 Set.of(CATALOG_NAME_PROPERTY, SCHEMA_NAME_PROPERTY, EXASOL_IMPORT_PROPERTY, EXASOL_CONNECTION_PROPERTY,
-                        EXASOL_IS_LOCAL_PROPERTY, IGNORE_ERRORS_PROPERTY), //
+                        EXASOL_IS_LOCAL_PROPERTY, IGNORE_ERRORS_PROPERTY, GENERATE_JDBC_DATATYPE_MAPPING_FOR_EXA), //
```

Comment:
After some thinking I propose to change the propertie's name to something that doesn't sound "boolean" but more like an enum, e.g. `IMPORT_FROM_EXA_DATATYPES` and values `EXPLICIT` and `OMIT`.